packages/web-features-figuring/src/browser-compat-data/browser.ts

```diff
@@ -20,11 +23,24 @@ export function browser(id: string, compat = defaultCompat): Browser {
 export class Browser {
   id: BrowserName;
   data: BrowserStatement;
-  _releases: Release[] | undefined;
+  releases: ReadonlyArray<Release>;
 
   constructor(id: BrowserName, data: BrowserStatement) {
     this.id = id;
     this.data = data;
+
+    const sortedReleaseData: [version: string, data: ReleaseStatement][] =
+      Object.entries(data.releases).sort(([a], [b]) => compareVersions(a, b));
+
+    const releases = sortedReleaseData.map(
+      ([version, data], index) => new Release(this, version, data, index),
+    );
```

Comment:
I'm going to leave this one for now. This is one of those places where TS makes the "simpler" thing more complex, because it can't follow unwinding of `version` from `data.releases`.